easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.2-binutils-2.25.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'binutils'
+version = '2.25'
+
+homepage = 'http://directory.fsf.org/project/binutils/'
+description = "binutils: GNU binary utilities"
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+toolchain = {'name': 'GCC', 'version': '4.9.2-binutils-%(version)s'}
+
+dependencies = [('zlib', '1.2.8')]
```

Comment:
I think we should consider statically linking zlib here too, to avoid pinning down the zlib version in the toolchain (there are pros and cons to hardwiring the zlib version to the toolchain)...